user_test.go

```diff
@@ -89,7 +89,7 @@ func TestFollowUsersMissingScope(t *testing.T) {
 	client := testClientString(http.StatusForbidden, json)
 	addDummyAuth(client)
 
-	err := client.Follow(ID("exampleuser01"))
+	err := client.FollowUser(ID("exampleuser01"))
```

Comment:
In the tests we should verify that the `type` is set correctly.

There should be a `getLastRequest ()` that will give you the HTTP request that was made, and then you can inspect its URL values at `req.URL.Query()`.